### ISSUE TITLE
Coordinate with kiwix to list our zims and repositories.

### ISSUE THREAD

**Issue by vvasuki** [sanskrit-coders/zim-sa#2]
# 

My query to kiwix about this view:
![image](https://cloud.githubusercontent.com/assets/2664797/13729960/603d7da2-e8fe-11e5-900b-3fcf11117cfd.png)
1. In Kiwix I do not see sanskrit wikisites. But we have made zimfiles of the sites separately. How do we get you make them available?
2. How do we become a publisher so that zimfiles from our project become available (nascent website - https://sites.google.com/site/sanskritcode/richer-sam-content/1-offline-web ) via the publisher dropdown in kiwix?
# 

This met with the following response
in this [FB thread](https://www.facebook.com/photo.php?fbid=10153572513822989&set=o.314470148640373&type=3&comment_id=10153577644802989&ref=notif&notif_t=photo_comment). They want to see our zim files. dAmodara, can you show them when they're ready?

**Comment by damooo** [comment]
yes. actually,  raaamayana is over tomarrow morning, i will post. and remaining too.. by evening

**Comment by damooo** [comment]
And Emmanuel, developer of kiwix also in contact at kiwix sourceforge page. and will post there too, about it.

**Comment by vvasuki** [comment]
Wonderful to note that so many zim files are ready! Can you work with Emmanuel to make it available on their repo?
